meroxa/resource_connector.go

```diff
@@ -133,17 +135,6 @@ func resourceConnectorCreate(ctx context.Context, d *schema.ResourceData, m inte
 		input.PipelineName = v.(string)
 	}
 
-	if v, ok := d.GetOk("config"); ok {
-		input.Configuration = v.(map[string]interface{})
-	}
-
-	if v, ok := d.GetOk("input"); ok {
```

Comment:
don't we still need input?

Reply:
Yes, we do, it is set when the config is set.